### DISCUSSION TITLE
Vue devtools not working (v5 or v6)

### DISCUSSION THREAD

**Discussion by ewanwalk** [quasarframework/quasar#10400] [General - Components / Directives / etc]
I've been trying to get the chrome extension vue-devtools to work on a fresh quasar v2 intstall however it appears nothing works. 

I've tried both version 5 and the beta version 6 both give varying errors:

V5
![v5](https://user-images.githubusercontent.com/8203640/129591978-690c1acf-c4bc-46e8-9bab-e47cc31269a1.png)

V6
![v6](https://user-images.githubusercontent.com/8203640/129591994-e7b52773-2b52-42f3-8a9b-170ab0bcbf8b.png)

It appears that V5 detects vue however does not work when you switch to the vue tab. Would anyone be able to point me in the right direction here? I believe this worked out of the box for me in v1 when I last tried quasar.

**Comment by MilosPaunovic** [comment]
Check this out:
https://github.com/quasarframework/quasar/discussions/10351#discussioncomment-1159828

**Reply by ewanwalk** [reply]
As my post states, I've tried both v5 and v6 which both do not work.

**Comment by pdanpdan** [comment]
3rd match on google, from vue's site:
https://v3.vuejs.org/guide/installation.html

**Reply by ewanwalk** [reply]
It is installed and setup correctly, it also works for every project except for a quasar based one. This likely means I have to do something else that isnt outlined anywhere in the quasar documentation.
